tests/locationConfig/locationConfigTests.json

```diff
@@ -109,5 +109,41 @@
             "bucketMatch": true,
             "azureContainerName": "s3createbucketonfly"
         }
+    },
+    "gcpbackend": {
+        "type": "gcp",
+        "legacyAwsBehavior": true,
+        "details": {
+            "gcpEndpoint": "storage.googleapis.com",
+            "bucketName": "zenko-gcp-bucket",
+            "mpuBucketName": "zenko-gcp-mpu",
+            "overflowBucketName": "zenko-gcp-overflow",
+            "bucketMatch": true,
+            "credentialsProfile": "google"
+        }
+    },
+    "gcpbackend2": {
```

Comment:
Can you add two more, one with proxy: true, https: true and the other with proxy: false and https: false.

Reply:
Will add the 2 new constraints